src/clients/java/src/client.zig

```diff
@@ -88,25 +88,18 @@ const ReflectionHelper = struct {
         request_end_request_method_id = null;
     }
 
-    pub fn throwInitializationException(env: *jui.JNIEnv, status: tb.tb_status_t) void {
+    pub fn throwInitializationException(env: *jui.JNIEnv, err: tb.InitError) void {
         assert(initialization_exception_class != null);
         assert(initialization_exception_ctor_id != null);
 
-        var exception = env.newObject(
-            initialization_exception_class,
-            initialization_exception_ctor_id,
-            &[_]jui.jvalue{jui.jvalue.toJValue(@bitCast(jui.jint, @enumToInt(status)))},
-        ) catch {
-            // It's unexpected here: we did not initialize correctly or the JVM is out of memory.
-            @panic("JNI: Unexpected error creating a new exception.");
+        var buf: [512]u8 = undefined;
+        var msg = std.fmt.bufPrintZ(&buf, "{s}", .{@errorName(err)}) catch {
```

Comment:
We can't do that, our `InitializationException` class expects an integer value on the constructor, that's why we don't use `env.throwNew` but instead we construct a new instance.

We should use the function `context_error_to_status` here, and keep passing the integer value to the Java side.